mixins/_visuallyhidden.scss

```diff
@@ -0,0 +1,16 @@
+//
+// Visually Hidden
+// --------------------
+//
+//
+
+@mixin visuallyhidden {
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  clip: rect(0 0 0 0);
+}
```

Comment:
To the debate mixins vs. extends there's a lot to say. Perhaps we should change sides an use mixins, I'm not 100% sure atm. There're some nice articles out there, which say everybody should:
- http://csswizardry.com/2014/11/when-to-use-extend-when-to-use-a-mixin/
- http://www.sitepoint.com/avoid-sass-extend/
- https://tech.bellycard.com/blog/sass-mixins-vs-extends-the-data/

But I also like this article where the power of extends is also shown (and yes, there're solutions to use placeholder-extends in media queries):
- http://www.smashingmagazine.com/2015/05/extending-in-sass-without-mess/